x-pack/test/osquery_cypress/fleet_server.ts

```diff
@@ -6,40 +6,52 @@
  */
 
 import { ToolingLog } from '@kbn/tooling-log';
-import execa from 'execa';
-import { runFleetServerIfNeeded } from '@kbn/security-solution-plugin/scripts/endpoint/endpoint_agent_runner/fleet_server';
+import { KbnClient } from '@kbn/test';
+import {
+  StartedFleetServer,
+  startFleetServer,
+} from '@kbn/security-solution-plugin/scripts/endpoint/common/fleet_server/fleet_server_services';
 import { Manager } from './resource_manager';
+import { getLatestAvailableAgentVersion } from './utils';
 
 export class FleetManager extends Manager {
-  private fleetContainerId?: string;
-  private log: ToolingLog;
+  private fleetServer: StartedFleetServer | undefined = undefined;
```

Comment:
FYI: In the future, we may want to consider removing this module and replacing it with `env.WITH_FLEET_SERVER` in the Cypress config file.